stable/elasticsearch/values.yaml

```diff
@@ -153,13 +153,13 @@ master:
     # requests:
     #   cpu: "25m"
     #   memory: "128Mi"
-  resources:
-    limits:
-      cpu: "1"
-      # memory: "1024Mi"
-    requests:
-      cpu: "25m"
-      memory: "512Mi"
+  resources: {}
```

Comment:
why do we need this comment here?

Reply:
For the reference on how to properly set the values for `resources` field.